jetbrains-core/resources/META-INF/plugin.xml

```diff
@@ -7,7 +7,19 @@
     <version>1.0</version>
 
     <description><![CDATA[
-      Plugin for interacting with AWS resources.
+<p>The AWS Toolkit is an open-source plug-in for JetBrains IDEs that makes it easier to develop applications built on AWS. The toolkit helps you create test, and debug serverless applications built using the AWS Serverless Application Model (<a href="https://aws.amazon.com/serverless/sam/">SAM</a>).
+</p>
+
+<p><h2><strong>Features</strong></h2></p>
+
+<ul>
+    <li><strong>New Project Wizard</strong> - Get started quickly by using one of the quickstart serverless application templates</li>
+    <li><strong>Run/Debug Local Lambda Functions</strong> - Locally test and step-through debug functions in a Lambda-like execution environment provided by the <a href="https://github.com/awslabs/aws-sam-cli">AWS SAM CLI</a></li>
```

Comment:
Says python Java only?

Reply:
I've manually updated it in the plugin repo so it's only available in pycharm & intellij